pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -180,6 +189,11 @@ func (h *NetworkUtilsHandler) HasNatIptables(proto iptables.Protocol) bool {
 	return true
 }
 
+func (h *NetworkUtilsHandler) ConfigureIpv4ArpIgnore() error {
+	err := sysctl.New().SetSysctl(sysctl.New().GetIpv4ArpIgnoreAll(), 1)
```

Comment:
This may be easier to read if you use a constant like we do for e.g. `systcl.NetIPv6Forwarding`.